server/src/cli.rs

```diff
@@ -39,6 +41,28 @@ pub struct EdgeArgs {
     /// Get data for these client tokens at startup. Hot starts your feature cache
     #[clap(short, long, env, value_delimiter = ',')]
     pub tokens: Vec<String>,
+
+    /// Expects curl header format (-H <HEADERNAME>: <HEADERVALUE>)
+    /// for instance `-H X-Api-Key: mysecretapikey`
+    #[clap(short = 'H', long, env, value_delimiter = ',', value_parser = string_to_header_tuple)]
+    pub custom_client_headers: Vec<(String, String)>,
+}
+
+pub fn string_to_header_tuple(s: &str) -> Result<(String, String), String> {
+    let format_message =
+        "Please pass headers in the format <headername>: <headervalue>".to_string();
```

Comment:
If I'm reading the code correctly, that whitespace is optional there
```suggestion
        "Please pass headers in the format <headername>:<headervalue>".to_string();
```